query-engine/query-engine-wasm/build.sh

```diff
@@ -13,12 +13,14 @@ OUT_NPM_NAME="@prisma/query-engine-wasm"
 if [[ -z "${WASM_BUILD_PROFILE:-}" ]]; then
     # use `wasm-pack build --release` by default on CI only
     if [[ -z "${BUILDKITE:-}" ]] && [[ -z "${GITHUB_ACTIONS:-}" ]]; then
-        WASM_BUILD_PROFILE="dev"
```

Comment:
Do we use this script to build the wasm engine even locally for running tests etc? If so, do you know how much slower the profile profile is compared to the dev one?

The reason I'm asking is that if it is much slower, then perhaps should be not make it the default to avoid slowing us down for the 99% case where we don't need to use twiggy to profile things?

Reply:
Fair enough, `dev` is the default one locally. I've also expanded the README.md to take into account how to customise build profiles, and I've added links to "tutorials" for `twiggy` and `./wasm-inspect.sh`.